src/components/Map.astro

```diff
@@ -6,10 +6,9 @@ const key = process.env.GOOGLE_API_KEY;
     iframe {
         border: 0;
         height: 0;
-    }
-
-    iframe[src] {
-        height: 400px;
+        grid-area: map;
+        height: 100%;
```

Comment:
when i inspect it shows the iframe is not super tall: 
![image](https://github.com/user-attachments/assets/c7cce36a-22a2-498a-9830-ac58ff459633)

but maybe itll look right?

you can make the map and the weather results wider too if it makes sense.